human_robot_collaboration/src/baxter_display.cpp

```diff
@@ -310,6 +341,19 @@ class BaxterDisplay
         ROS_INFO_COND(print_level>=1, "Ready!!");
     };
 
+    void imageLoad(void)
```

Comment:
add documentation for this function

Reply:
In general, I don't think I would add a function for this. I would move everything into `createBtmImage` for simplicity.